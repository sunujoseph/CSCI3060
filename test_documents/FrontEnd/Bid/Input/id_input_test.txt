login
username
bid
sellername
itemname
100
logout