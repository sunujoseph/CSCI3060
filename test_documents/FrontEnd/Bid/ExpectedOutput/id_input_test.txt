Enter user name: 
User name accepted: 
Enter seller's username: 
Enter item name: 
Current highest bid is: $10.00
Enter Bid: 
Bid accepted.
Logged out.
